Rules/CommonScripts/Holiday/Christmas/Christmas.as

```diff
@@ -54,8 +56,7 @@ void onTick(CRules@ this)
 		}
 	}
 	
-	
-	if (!isServer() || this.isWarmup() || !(this.gamemode_name == "CTF" || this.gamemode_name == "TTH" || this.gamemode_name == "SmallCTF"))
+	if (!isClient() || this.isWarmup() || !(this.gamemode_name == "CTF" || this.gamemode_name == "TTH" || this.gamemode_name == "SmallCTF"))
```

Comment:
accidental change? spawning presents/blobs etc should be on serwer

Reply:
ok